lib/wpcom-error-handler/wpcom-error-handler.php

```diff
@@ -155,29 +171,32 @@ function wpcom_custom_error_handler( $whether_i_may_die, $type, $message, $file,
 		}
 
 		if ( 'stderr' === $display_errors && defined( 'STDERR' ) && is_resource( STDERR ) ) {
-			fwrite( STDERR, sprintf( $display_errors_format, $string, $message, $file, $line, htmlspecialchars( $source ), htmlspecialchars( $backtrace ) ) );
+			fprintf( STDERR, $display_errors_format, $string, esc_html( $message ), esc_html( $file ), esc_html( $line ), esc_html( $source ), esc_html( $backtrace ) );
```

Comment:
> File system operations only work on the `/tmp/` and `wp-content/uploads/` directories. To avoid unexpected results, please use helper functions like `get_temp_dir()`  or `wp_get_upload_dir()` to get the proper directory path when using functions such as fwrite(). For more details, please see: https://docs.wpvip.com/technical-references/vip-go-files-system/local-file-operations PHPCS(WordPressVIPMinimum.Functions.RestrictedFunctions.file_ops_fwrite)